automatic/autohotkey.install/tools/chocolateyInstall.ps1

```diff
@@ -10,12 +12,44 @@ if (!$pp.DefaultVer){
 $packageArgs = @{
   packageName    = 'autohotkey.install'
   fileType       = 'exe'
-  file           = Get-Item "$toolsDir\*.exe"
-  silentArgs     = "/S /$($pp.DefaultVer)"
+  file           = Join-Path -Path $toolsDir -ChildPath $installerv2File
+  silentArgs     = "/silent"
   softwareName   = 'AutoHotkey*'
   validExitCodes = @(0, 1223)
 }
-Install-ChocolateyInstallPackage @packageArgs
+
+Install-ChocolateyInstallPackage @packageArgsv2
```

Comment:
This one is the cause of the failure.

You are attempting to use the variable `packageArgsv2`, however it is still called `packageArgs` above.

Reply:
I've pushed a fixed version.